lib/yalphabetize/logger.rb

```diff
@@ -58,7 +58,7 @@ def puts_offence(file_path, status)
     end
 
     def list_offences_color
-      offences? ? :red : :green
+      uncorrected_offences? ? :red : :green
```

Comment:
No problem.  Since this seems to be the only place where we're checking for _any_ offences, I'm just using `any?` directly, rather than re-introducing the helper method.